habitat_baselines/common/env_utils.py

```diff
@@ -0,0 +1,141 @@
+import random
+
+import numpy as np
+
+import habitat
+from habitat import Config, SimulatorActions, make_dataset
+
+
+class NavRLEnv(habitat.RLEnv):
+    def __init__(self, config_env, config_baseline, dataset):
```

Comment:
Comments here would be helpful to explain more about the parameters.

Reply:
Also, it is unnatural to write it as `config_env` and `config_baseline` instead of `env_config` and `baseline_config`.

Reply:
And instead of `baseline_config`, it should be `model_config` now.

Reply:
Also, why this isn't single config instead of being `env_config` separately and `baseline_config` separately.

Reply:
@apsdehal This naming choices are largely inherited from the original [`train_ppo.py`](https://github.com/facebookresearch/habitat-api/blob/782f1f9aec09895dc27347927f01998ba383b8b4/habitat_baselines/train_ppo.py#L25). I do have some alternatives in mind (see [comments here](https://github.com/facebookresearch/habitat-api/pull/153#discussion_r302721173)). Also, as to why it's not a single config yet: I definitely intend to merge two configs (see [here](https://github.com/JasonJiazhiZhang/habitat-api/blob/f6478167b3e46b897e4b7b1158b265b63a3b6c31/habitat_baselines/common/env_utils.py#L76) and [here](https://github.com/JasonJiazhiZhang/habitat-api/blob/f6478167b3e46b897e4b7b1158b265b63a3b6c31/habitat_baselines/common/utils.py#L90-L97), comparing to [this](https://github.com/JasonJiazhiZhang/habitat-api/blob/782f1f9aec09895dc27347927f01998ba383b8b4/habitat_baselines/train_ppo.py#L92)), and should modify NavRLEnv for use one config for sure. Thank you for catching this 👍 .  I'll make those changes to NavRLEnv to  take one unified config, if people agree with this change.